### PR TITLE
Move core change 65 to 69 in Ironwood

### PR DESCRIPTION
**Pull request by zeshanarifios**

### CODE REVIEW

File: lms/templates/dashboard.html [hunk 1]
```diff
@@ -49,31 +47,8 @@
...
-        dashboardPath: "${reverse('dashboard') | n, js_escaped_string}",
-        signInPath: "${reverse('signin_user') | n, js_escaped_string}",
-        browseCourses: "${marketing_link('COURSES') | n, js_escaped_string}",
-        isEdx: false
-      });
-    });
-  </%static:webpack>
   % if settings.FEATURES.get('ENABLE_DASHBOARD_SEARCH'):
-    <%static:require_module module_name="course_search/js/dashboard_search_factory" class_name="DashboardSearchFactory">
+    <%static:require_module module_name="js/search/dashboard/dashboard_search_factory" class_name="DashboardSearchFactory">
```
Comment (ahmed-faraz46) [inline lms/templates/dashboard.html]: Let's please make sure this module exists at the mentioned path, becuase we've pasted the whole file from ficus, so we should verify all the paths

**Comment by zeshanarifios** [comment]
@ahmed-faraz46 please have a look again.

**Comment by ahmed-faraz46** [comment]
@zeshanarifios for changed 68 (` lms/templates/student_account/login_and_register.html Just indentation of this file is changed. But it’s incorrect we should keep edX’s indentation (We kept edX’s indentation)`) please make sure that the code of this file is not modified by ironwood.

**Comment by zeshanarifios** [comment]
@ahmed-faraz46 In this ticket we are supposed to move core changes of Philu from ficus to ironwood. In ficus there were just indentation changes, That's why we skip this core change as it can be avoided. For now it's fine.

**Pull request merged**